.github/workflows/release_pypi.yml

```diff
@@ -0,0 +1,104 @@
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Install maturin
+        run: pip install maturin
+      - name: Build wheels - x86_64
+        run: |
+          maturin build -m pyoxidizer/Cargo.toml -b bin --target x86_64-apple-darwin --release --out dist --no-sdist
+          pip install pyoxidizer --no-index --find-links dist --force-reinstall
+      - name: Build wheels - universal2
+        env:
+          DEVELOPER_DIR: /Applications/Xcode.app/Contents/Developer
+          MACOSX_DEPLOYMENT_TARGET: '10.9'
+        run: |
+          # set SDKROOT for C dependencies like bzip2
+          export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
+          maturin build -m pyoxidizer/Cargo.toml -b bin --release --universal2 --out dist --no-sdist
+          pip install pyoxidizer --no-index --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        platform: [
+          { python-architecture: "x64", target: "x86_64-pc-windows-msvc" },
+          { python-architecture: "x86", target: "i686-pc-windows-msvc" },
+        ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+          architecture: ${{ matrix.platform.python-architecture }}
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: ${{ matrix.platform.target }}
+          profile: minimal
+          default: true
+      - name: Install maturin
+        run: pip install --pre maturin
+      - name: Build wheels
+        run: |
+          maturin build -m pyoxidizer/Cargo.toml -b bin --release --out dist --no-sdist --target ${{ matrix.platform.target }}
+          pip install pyoxidizer --no-index --find-links dist --force-reinstall
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [
+          { manylinux: '2010', target: "x86_64-unknown-linux-musl", image_tag: "x86_64-musl" },
+          { manylinux: '2010', target: "i686-unknown-linux-musl", image_tag: "i686-musl" },
+          { manylinux: '2014', target: "aarch64-unknown-linux-musl", image_tag: "aarch64-musl" },
+          { manylinux: '2014', target: "armv7-unknown-linux-musleabihf", image_tag: "armv7-musleabihf" },
```

Comment:
Do we need the 2014 variant? The year component in the `manylinux` platform tag mainly denotes symbol compatibility with glibc. Since we statically link with musl, we should run on all Linux, so the 2014 variant probably isn't needed.

Reply:
It's for aarch64 and armv7 Linux which are only available since manylinux2014. https://www.python.org/dev/peps/pep-0599/

We can remove it if pyoxidizer doesn't support architectures other that x86_64/i686.